packages/docs/type-test-arabic.md

```diff
@@ -0,0 +1,243 @@
+---
+template: plain
+id: base-type
+title: Type Examples
+headline: Type Examples
+lede: A comprehensive set of mixed-content, in-situ type specimens for Arabic.
+description:
+---
+<div dir="rtl">
+
+# هذا هو عنوان المستوى الأول
+
+يقوم Vangelis بإشعال الطاقة المخبأة في المادة النجمية tesseract التي تحصد ضوء النجوم وتحافظ على تلك المستعمرات ذات النقاط الزرقاء الباهتة وتعتز بها. مواطنون محدودون ولكن غير مقيدون من عصور بعيدة ، مواطنون من حقب بعيدة ، أثارهم مواطنو النجوم من عصور بعيدة بقصص جميلة لا يوجد دليل جيد عليها.
```

Comment:
I know I'm the one who suggested this, but I'm wondering if this may be better served as a story of some kind, that way we can test it using VRT longer term.